### PR TITLE
Update nagios tutorial for cli

### PR DESCRIPTION
**Pull request by hkariti**
@shaharke Is that enough?

### CODE REVIEW

**Comment by erikzaadi** [comment]
http://bp-docs-staging.s3-website-us-west-1.amazonaws.com/feature/agent_cli/integrations/index.html#nagios--debian-

**Comment by hkariti** [comment]
Thanks for that. I'm thinking about separating the permissions part to a different step, because ppl might miss it.

Also, the output example I give is missing the defaults in square brackets for each value. It actually should look like this:

    Log File Path [/usr/local/nagios/var/nagios.log]: /var/log/nagios3/nagios.log
    Status Dat Path [/usr/local/nagios/var/status.dat]: /var/cache/nagios3/status.dat
    Objects Cache Path [/usr/local/nagios/var/objects.cache]: /var/cache/nagios3/objects.cache

I removed it because I thought it'd look clearer without having two paths in the example.

Thoughts?

**Comment by erikzaadi** [comment]
IMHO since it's a code block it should be as the real output is :ghost:

**Comment by shaharke** [comment]
I actually agree with Hagai's intuitions:

   1. No default paths
   2. Separate permissions

Shahar

**Comment by hkariti** [comment]
Now with zenoss. Still missing zabbix. @iravid - is it in your branch?

**Comment by iravid** [comment]
aye

On Tue, Apr 21, 2015 at 11:57 AM, Hagai Kariti <notifications@github.com>
wrote:

> Now with zenoss. Still missing zabbix. @iravid <https://github.com/iravid>
> - is it in your branch?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bigpandaio/docs.bigpanda.io/pull/6#issuecomment-94710206>
> .
>



-- 
Itamar

**Comment by hkariti** [comment]
pull request away!

File: _integrations/agent-common/configure-agent/zabbix.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+Execute the following command to initially configure the agent:
+
+    $ sudo bigpanda-config --init --token $TOKEN
+
+<!-- section-separator -->
+
+#### Configure Zabbix
+Execute the following command and follow the on-screen instructions to configure the BigPanda Zabbix plugin: 
+
+    $ sudo bigpanda-config --add zabbix --app-key $STREAM_ID
```
Comment (hkariti) [inline _integrations/agent-common/configure-agent/zabbix.md]: forgot bigpanda-config --init --token $API_TOKEN

**Pull request merged**